python/runtime/pai/train.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2020 The SQLFlow Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
```

Comment:
rename this file to train_tensorflow.py?

Reply:
I think we can keep the name `train.py`, then we can call the train function with the format `runtime.pai.train` and `runtime.local.train` and `runtime.elasticdl.train` etc.